configs/avae-test-config.yml

```diff
@@ -25,7 +25,7 @@ freq_sta: 10  # frequency of state saves
 
 #### Loss parameters
 
-beta:  1
+beta_max:  1
```

Comment:
Do we only need one of these arguments here in the config? What about `beta_min`, `beta_cycle` etc?